test/disabled/connection.test.js

```diff
@@ -19,7 +18,6 @@ describe('Connection', function() {
           noop,
           Object.assign(
```

Comment:
nit: this can probably be make into one line now (eslint/prettier doesn't catch it because its still technically correct):
```
Object.assign({ port: server.port }, config)
```